cs_test/NIPS.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using VW;
+using VW.Labels;
+using VW.Serializer.Attributes;
+
+namespace cs_test
+{
+    public class NIPS
+    {
```

Comment:
Is NIPS here referring to the conference? If so can you rename it to something else more relevant